.github/workflows/lock-issues.yml

```diff
@@ -0,0 +1,15 @@
+me: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
```

Comment:
Actually one more thought while we're here. It looks like some other projects have [shortened the window](https://github.com/pyca/pyopenssl/blob/main/.github/workflows/lock.yml#L13-L14) from 365 days to 90. That may be more reasonable to prevent unrelated pings resetting such a long timer.